src/core/renderController.ts

```diff
@@ -91,34 +91,23 @@ export module Core {
     export function flush() {
       if (_animationRequested) {
         // Layout
-        var toCompute = d3.values(_componentsNeedingComputeLayout);
-        toCompute.forEach((c) => c.computeLayout());
+        _componentsNeedingComputeLayout.values().forEach((c: Component) => c.computeLayout());
```

Comment:
If you're worried about character constraints, feel free to remove the typing at the function declaration stage.  We should probably do this moving forward